runtime/Cargo.toml

```diff
@@ -22,6 +22,7 @@ dashmap = { workspace = true, features = ["rayon", "raw-api"] }
 dir-diff = { workspace = true }
 flate2 = { workspace = true }
 fnv = { workspace = true }
+fs-err = "2.9.0"
```

Comment:
this should be added in the root Cargo.toml, and the workspace version used here